pkg/cli-runtime/config_test.go

```diff
@@ -153,40 +153,34 @@ func TestConfig_Print(t *testing.T) {
 		})
 	}
 }
-func TestConfig_EmojiPrint(t *testing.T) {
+
+func TestConfig_Emoji(t *testing.T) {
 	scheme := runtime.NewScheme()
 	config := cli.NewDefaultConfig("test", scheme)
 
 	tests := []struct {
 		name    string
 		icon    cli.Icon
-		args    []interface{}
-		printer func(icon cli.Icon, format string, a ...interface{}) (n int, err error)
-		stdout  string
+		noColor bool
+		input   string
+		output  string
 	}{{
-		name:    "EmojiSuccessf",
-		icon:    cli.ThumpsUp,
-		args:    []interface{}{"Pods created Successfully"},
-		printer: config.EmojiSuccessf,
-		stdout:  `👍 Pods created Successfully`,
+		name:   "Text with emoji",
+		icon:   cli.FloppyDisk,
+		input:  "Source",
+		output: `💾 Source`,
 	}, {
-		name:    "EmojiBoldf",
-		icon:    cli.FloppyDisk,
-		args:    []interface{}{"Source"},
-		printer: config.EmojiBoldf,
-		stdout:  `💾 Source`,
+		name:    "Text without emoji",
+		noColor: true,
```

Comment:
We would need an icon too part of the test to make sure the noColor flag working